<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomationDetailPage</class>
 <widget class="QWidget" name="AutomationDetailPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>307</width>
         <height>34</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
border: none;
font-family: Open Sans;
font-size: 24px;
line-height: 34px;
letter-spacing: 0%;
font-weight: 600;</string>
       </property>
       <property name="text">
        <string>Lorem ipsum nibh lacus</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Resources/Resources.qrc">
         <normaloff>:/images/nav_back.svg</normaloff>:/images/nav_back.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="editAutomationButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>178</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>178</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
border: 1.5px solid #1a1a1a;
border-radius: 8px;
color: #1a1a1a;
font-family: Open Sans;
font-size: 18px;
line-height: 24px;
font-weight: 500;</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testAutomationButton">
       <property name="minimumSize">
        <size>
         <width>178</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>178</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FBAE17;
border: none;
border-radius: 8px;
color: white;
font-family: Open Sans;
font-size: 18px;
line-height: 24px;
font-weight: 500;</string>
       </property>
       <property name="text">
        <string>Test Automation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="propertyLayout">
     <property name="spacing">
      <number>210</number>
     </property>
     <item>
      <widget class="AutomationCellProperty" name="aiModel" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>144</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AutomationCellProperty" name="version" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AutomationCellProperty" name="trigger" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="mainPanelLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="mainPanelTitleBarLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
border: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../../Resources/Resources.qrc">
           <normaloff>:/images/Automations/prompt.svg</normaloff>:/images/Automations/prompt.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-family: Open Sans;
font-size: 20px;
font-weight: 600;
line-height: 28px;
letter-spacing: 0%;
color: #1a1a1a;</string>
         </property>
         <property name="text">
          <string>Prompt</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="promptBrowser">
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
    border: none; /* Remove border */
    background-color: white;
    padding: 8px;
    font-family: Open Sans;
    font-size: 18px;
}

/* Scrollbar Customization */
QScrollBar:vertical {
    background: #F0F0F0;
    width: 6px;
    margin: 0px;
    border-radius: 3px;
}
QScrollBar::handle:vertical {
    background: #2D1457;
    min-height: 30px;
    border-radius: 3px;
}
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: none;
}

QScrollBar:horizontal {
    background: #F0F0F0;
    height: 6px;
    margin: 0px;
    border-radius: 3px;
}
QScrollBar::handle:horizontal {
    background: #2D1457;
    min-width: 30px;
    border-radius: 3px;
}
QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal,
QScrollBar::add-page:horizontal,
QScrollBar::sub-page:horizontal {
    background: none;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutomationCellProperty</class>
   <extends>QWidget</extends>
   <header location="global">automationcellproperty.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
